website/pages/playground/index.html

```diff
@@ -339,58 +357,14 @@ <h1>Prettier <span id="version"></span></h1>
 
   <div class="bottom-bar">
     <div class="bottom-bar-buttons">
+      <button type="button" class="btn" id="button-options">
+        See options
```

Comment:
`Show options`?

Reply:
That copy is better.. let's change that as well.